### PR TITLE
CernVM-FS 2022 project proposal

### PR DESCRIPTION
**Pull request by radupopescu**

### CODE REVIEW

File: _gsocproposals/2022/proposal_CernVM-FS-firecracker-based-integration-tests.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+---
+title: CernVM-FS integration test framework powered by Firecracker VM
+layout: gsoc_proposal
+project: CernVM
```
Comment (aflp91) [inline _gsocproposals/2022/proposal_CernVM-FS-firecracker-based-integration-tests.md:4]: You have to add a `CernVM` project in `_gsocprojects` directory. (But last year, you had a `CernVM-FS` project !)

Comment (aflp91) [inline _gsocproposals/2022/proposal_CernVM-FS-firecracker-based-integration-tests.md:4]: Project difficulty, duration and mentor availability required:
difficulty: [medium or high]
duration: [350 or 175 hours]
mentor_avail: [month interval]

File: _gsocproposals/2022/proposal_CernVM-FS-firecracker-based-integration-tests.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+## Expected results
+
+A usable system for running CernVM-FS integration tests with isolation between
+test cases, powered by Firecracker VM, and running on CERN Openstack
+infrastructure.
+
+## Requirements
+Linux, shell scripting, basic networking, VMs, containers
+
+## Mentors
```
Comment (aflp91) [inline _gsocproposals/2022/proposal_CernVM-FS-firecracker-based-integration-tests.md:51]: Affiliation needed; main mentor in bold. Please add Radu in `2022/mentors.md`

File: _gsocproposals/2022/proposal_CernVM-FS-firecracker-based-integration-tests.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+isolation) and usability (run integration tests easily on a development
+machine).
+
+## Task outline
+* Develop a tool to produce a VM image in QCOW2 format for a specific Linux
+  distribution, containing an installation of the desired CernVM-FS version or
+  tag (ex: Ubuntu 20.04 + CernVM-FS 2.9.1).
+* Use the produced VM image with Firecracker VM on a Linux host and run a single
+  integration test case, powering down the VM after the test case was run;
+  changes to the VM image occuring during the test should not be persisted.
```
Comment (aflp91) [inline _gsocproposals/2022/proposal_CernVM-FS-firecracker-based-integration-tests.md:35]: `occuring` =>  `occurring`

**Review by aflp91 — CHANGES_REQUESTED** [review]
See my comments added to file.

File: _gsocproposals/2022/proposal_CernVM-FS-firecracker-based-integration-tests.md (hunk 1 above)
Comment (radupopescu) [reply _gsocproposals/2022/proposal_CernVM-FS-firecracker-based-integration-tests.md]: My mistake, it should be CernVM-FS. I copied the project file from 2021.

**Comment by radupopescu** [comment]
Thanks for the feedback! Hopefully, I implemented all the needed changes.

**Review by hegner — APPROVED** [review]

**Pull request merged**